reconcile/utils/git.py

```diff
@@ -25,3 +25,10 @@ def checkout(commit, wd):
     proc.communicate()
     if proc.returncode != 0:
         raise GitError('git checkout failed: {}'.format(commit))
+
+
+def is_file_in_git_repo(file_path):
+    real_path = os.path.realpath(file_path)
+    dir_path = os.path.dirname(real_path)
+    git_path = os.path.join(dir_path, '.git')
+    return os.path.exists(git_path)
```

Comment:
this only catches paths that are in the root of qontract-reconcile but not ones in subdirectories.

Reply:
fixed in https://github.com/app-sre/qontract-reconcile/pull/2066/commits/a5e879e254ea434254c7b71f667a840b2aa87cd5